.github/workflows/reusable_receptorctl_tests.yml

```
@@ -0,0 +1,54 @@
+---
+name: receptorctl
+
+"on":
+  workflow_call:
+
+jobs:
+  receptorctl:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        matrix:
+          python-version:
+          # NOTE: The highest and the lowest versions come
+          # NOTE: first as their statuses are most likely to
+          # NOTE: signal problems early
+          - 3.11
+          - 3.8
+          - >-
+            3.10
+          - 3.9
```

I added a nested matrix by accident the the previous snippet. This should fix it.
```suggestion
        python-version:
        # NOTE: The highest and the lowest versions come
        # NOTE: first as their statuses are most likely to
        # NOTE: signal problems early
        - 3.11
        - 3.8
        - >-
          3.10
        - 3.9
```